### PR TITLE
Release_2020_02_06

### PR DESCRIPTION
**Pull request by lucendio**
## New features

- SCIM top level extra attrs / rich info (#931)
  - Added to all endpoints under "/scim/v2"
- Create endpoint for default SSO code (#954)
  - New public endpoint:
    - GET "/sso/settings"
  - New private endpoint:
    - PUT "/i/sso/settings"

## Relevant for client developers

- add docs for default sso code (#960)
- Add missing options to services-demo config files (#961)

## Security fixes

- Remove verifcation code from email subject line. (#950)

## Internal changes

- Whitespace (#957)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
+# 2020-02-06
+
+## New features
+
+- SCIM top level extra attrs / rich info (#931)
+- Create endpoint for default SSO code (#954)
```
Comment (tiago-loureiro) [inline CHANGELOG.md]:
```suggestion
- Create endpoint for default SSO code (#954)
  - New public endpoint:
    - GET "/sso/settings"
  - New private endpoint:
    - PUT "/i/sso/settings"
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,24 @@
+# 2020-02-06
+
+## New features
+
+- SCIM top level extra attrs / rich info (#931)
```
Comment (tiago-loureiro) [inline CHANGELOG.md]:
```suggestion
- SCIM top level extra attrs / rich info (#931)
  - Added to all endpoints under "/scim/v2"
```

**Comment by tiago-loureiro** [comment]
This should be against `master`, not `develop`

**Comment by tiago-loureiro** [comment]
> This should be against `master`, not `develop`

Fixed, thanks @lucendio !

File: CHANGELOG.md (hunk 2 above)
Comment (lucendio) [reply CHANGELOG.md]: no nitpicking at all valuable information. solved.

**Pull request merged**